validator/src/bootstrap.rs

```diff
@@ -1437,6 +1437,10 @@ mod with_incremental_snapshots {
             }
         }
 
+        info!("Delete snapshot local file {:?}", snapshot_archives_dir);
```

Comment:
For user-facing message, prefer `Display` over `Debug`. For here, do `snapshot_archives_dir.display()` and use `{}`.

https://github.com/solana-labs/solana/blob/09b58e1cfbb7b568e8b4b677a889da96163ea38f/runtime/src/snapshot_utils.rs#L1264-L1267